appveyor.yml

```diff
@@ -1,8 +1,12 @@
-before_build:
+install:
+  - npm install -g mocha
   - npm install
+  - npm link
```

Comment:
Fixed by `1ac07d5`, effectively renamed mocha-appveyor.js -> mocha-appveyor-reporter.js to be consistent with the module name that way it can be used the same both locally and globally.